<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassSettingWindow</class>
 <widget class="QMainWindow" name="ClassSettingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>班级设置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../rc.qrc">
    <normaloff>:/ico/images/appstu.ICO</normaloff>:/ico/images/appstu.ICO</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tbClass">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>331</width>
      <height>251</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
    </property>
    <column>
     <property name="text">
      <string>班级编号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>专业名称</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>班级名称</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,4,1,4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>班级编号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editID"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>班级名称：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editName"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>340</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnAdd">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEdit">
       <property name="text">
        <string>修改</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDel">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExit">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>75</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>选择专业：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cboxGrade">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>138</width>
      <height>19</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../rc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>editID</sender>
   <signal>returnPressed()</signal>
   <receiver>editName</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editName</sender>
   <signal>returnPressed()</signal>
   <receiver>editID</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnExit</sender>
   <signal>clicked()</signal>
   <receiver>ClassSettingWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
